app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntrySheet.kt

```diff
@@ -139,6 +140,8 @@ class CustomDrugEntrySheet : BaseBottomSheet<
       if (result is Succeeded)
         hotEvents.onNext(FrequencyEdited(SelectDrugFrequencyDialog.readDrugFrequency(result)))
     }
+
+    drugDosageEditText.showKeyboard()
```

Comment:
What’s happening currently is,
Steps : when the sheet is opened and we show the keyboard --> we type the dosage in mg --> and I was stuck on the screen for a while until I understood to go back and click on done to save the dosage. Can you also close the keyboard on ime done action or check with the designers on the same?